tests/test_stp_global.rb

```diff
@@ -0,0 +1,525 @@
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/stp_global'
+
+# TestX__CLASS_NAME__X - Minitest for X__CLASS_NAME__X node utility class
+class TestStpGlobal < CiscoTestCase
+  # TESTS
+
+  @@clean = false # rubocop:disable Style/ClassVars
+  def setup
+    super
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+    @intf = Interface.new(interfaces[0])
+
+    # Only pre-clean interface on initial setup
+    config("default interface #{@intf}") unless @@clean
+    @@clean = true # rubocop:disable Style/ClassVars
+  end
+
+  def teardown
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+    super
+  end
+
+  def n7k_platform?
+    /N7/ =~ node.product_id
+  end
+
+  def n9k_platform?
+    /N(3|9)/ =~ node.product_id
+  end
+
+  def n6k_platform?
+    /N(5|6)/ =~ node.product_id
+  end
+
+  def test_bd_forward_time_change
+    skip('Platform does not support this property') if n6k_platform? ||
```

Comment:
Now that grpc is merged into develop let's remove the explicit skips.

 Our new process will be to explicitly do the following for platforms in the _exclude yaml entry.

 - Test that calls to config_get and config_set result in a Cisco::UnsupportedError being raised.
 - Test the calls to config_get_default for the property return nil.
 - Example: https://github.com/cisco/cisco-network-node-utils/blob/feature/grpc/tests/test_router_bgp.rb#L471